### PR TITLE
Bug 1824468: Add capacity for Local volume shows NaN TiB

### PR DESCRIPTION
**Pull request by cloudbehl**
**Before**
Screenshot 2020-04-15 at 6 10 51 PM

**After**
Screenshot 2020-04-15 at 6 08 28 PM

Signed-off-by: Ankush Behl <cloudbehl@gmail.com>

### CODE REVIEW

**Comment by gnehapk** [comment]
Please add the bugzilla id

File: frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx [hunk 1]
```diff
@@ -94,23 +94,26 @@ export const AddCapacityModal = withHandlePromise((props: AddCapacityModalProps)
...
-            </div>
-          </div>
+          {osdSizeWithoutUnit && (
+            <>
+              <label className="control-label" htmlFor="requestSize">
+                Raw Capacity
+                <FieldLevelHelp>{labelTooltip}</FieldLevelHelp>
+              </label>
+              <div className="ceph-add-capacity__form">
+                <input
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx:104]: Can we add a data-test-id to this input?

Comment (gnehapk) [reply frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx:104]: Its getting added in my PR. So, if we want, we can skip it here.

Comment (gnehapk) [reply frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx:104]: https://github.com/openshift/console/pull/5016/files#diff-ecff7031320d48d9e801ba99a2fb7188R123

Comment (cloudbehl) [reply frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx:104]: 👍

**Comment by cloudbehl** [comment]
This fix is for OCS 4.4 and in 4.5. This is anyways hidden & the right information is being shown.

**Comment by cloudbehl** [comment]
Closing this PR. WIll be fixed in 4.5!

**Pull request closed without merging**